manifests/service.pp

```diff
@@ -18,6 +18,8 @@
   }
 
   if $apache {
+    include ::apache
```

Comment:
This feels weird to me. I'd expect this to be set in config already.

Reply:
Trying to remember why this had to be there.